ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -192,6 +192,69 @@ private[testtool] abstract class CommandDeduplicationBase(
     }
   )
 
+  // Without the submission id we cannot assert the received completions for parallel submissions
+  if (deduplicationFeatures.appendOnlySchema)
+    testGivenAllParticipants(
+      s"${testNamingPrefix}SimpleDeduplicationMixedClients",
+      "Deduplicate commands within the deduplication time window using the command client and the command submission client",
+      allocate(SingleParty),
+    )(implicit ec =>
+      configuredParticipants => { case Participants(Participant(ledger, party)) =>
+        def combinations(elements: List[List[Boolean]]): List[List[Boolean]] = elements match {
```

Comment:
🚲 IIRC, this is a variation with repetitions, so to be more precise I would call it `variations`.